factory-layout/src/main/java/com/accesspoint/factorylayout/controller/LayoutController.java

```diff
@@ -0,0 +1,21 @@
+package com.accesspoint.factorylayout.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import com.accesspoint.factorylayout.Layout;
+import com.accesspoint.factorylayout.repository.LayoutRepository;
+
+@Controller
+@RequestMapping("/api/layouts")
+public class LayoutController{
+    @Autowired
+    LayoutRepository layoutRepository;
+
+    @PostMapping("/")
+    public Layout createLayout(@RequestBody Layout layout) {
+        return layoutRepository.save(layout);
```

Comment:
The mapping for the POST request looks to be right.

_I will let someone else review as well just in case I overlooked something._

Reply:
One thing That would be probably good to do is change the / on the post mapping to /save so something like that